static/babelrc.json

```diff
@@ -1,7 +1,12 @@
 {
-  "breakConfig": true,
   "sourceMap": "inline",
-  "blacklist": ["es6.forOf", "useStrict"],
-  "optional": ["asyncToGenerator"],
-  "stage": 0
+  "plugins": [
```

Comment:
We previously supported legacy decorators, right? I think we'd need to include that plugin.